docs/DEVELOPING.md

```diff
@@ -343,6 +343,139 @@ $ make ftest NAME=azure_blob_storage
 
 ℹ️ The e2e test uses default values defined in [configure Azure Blob Storage connector](#configure-azure-blob-storage-connector)
 
+
+## Oracle Database Connector
+
+The [Elastic Oracle Database connector](https://github.com/elastic/connectors-python/blob/8.7/connectors/sources/oracle.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
```

Comment:
We should also include the versions and editions of Oracle database we intend to support.